pgjdbc/src/test/java/org/postgresql/test/jdbc4/jdbc41/GetObjectTest.java

```diff
@@ -694,6 +694,39 @@ void getBlob() throws SQLException {
     }
   }
 
+  /**
+   * Test the behavior getObject for bytea columns.
+   */
+  @Test
+  void getBytea() throws SQLException {
+    Statement stmt = conn.createStatement();
+    conn.setAutoCommit(false);
+    try {
+      byte[] data = new byte[]{(byte) 0xDE, (byte) 0xAD, (byte) 0xBE, (byte) 0xEF};
+      PreparedStatement insertPS = conn.prepareStatement(TestUtil.insertSQL("table1", "bytea_column", "?"));
+      try {
+        insertPS.setBytes(1, data);
+        insertPS.executeUpdate();
+      } finally {
+        insertPS.close();
```

Comment:
Yes please use try with resources. We are now trying to move all new code to that style